src/include/api/vamana_index.h

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
(Minor) Can you remove this newline change, and the two right below it in this file?